css/at-rules/custom-media.json

```diff
@@ -0,0 +1,47 @@
+{
+  "css": {
+    "at-rules": {
+      "custom-media": {
+        "__compat": {
+          "description": "`@custom-media`",
+          "spec_url": "https://drafts.csswg.org/mediaqueries-5/#at-ruledef-custom-media",
+          "support": {
+            "chrome": {
+              "version_added": false,
+              "impl_url": "https://crbug.com/40781325"
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "146",
```

Comment:
Does this really work? I tested it and it didn't work.

```html
data:text/html;charset=UTF-8,<!DOCTYPE html>
<style>
@custom-media --foo (width > 0px);

@media (--foo) { 
  div {
    background: green;
    width: 200px;
    height: 200px;
  }
}
</style>
<div>This should be a green background.</div>
```

<img width="1510" height="804" alt="Image" src="https://github.com/user-attachments/assets/b533fb68-c4d7-4a62-a14d-351992a391c8" />

Reply:
@emilio Given [bug 2004653](https://bugzilla.mozilla.org/show_bug.cgi?id=2004653) (landed in Firefox 148), this feature actually doesn't work in Firefox 146. Will you backport it to Firefox 146?

Otherwise, all relevant MDN docs must be modified.

Reply:
@yisibl Since the feature is behind a flag, the fix probably won't be backported. If you like, you can open a PR correcting the version number, but once the feature ships in Firefox Stable, the flag entry will be removed anyhow.